TheJumpCloud/jumpcloud-ADMU
# Sa 2817 admu error messages

## Issues
* [SA-2817](https://jumpcloud.atlassian.net/browse/SA-2817) - Better Logging & Updated Prerequisite Paths

## What does this solve?

Systems downloading the JumpCloud agent prerequisites would typically see an error as the JumpCloud agent would begin downloading the agent prerequisite files: ('vc_redist.x64.exe', 'vc_redist.x86.exe')

## Is there anything particularly tricky?

## How should this be tested?

- On a windows VM bound to Azure AD, have at least one AD user on the system
- Sign in as a local administrator
- Pull this branch and run the ADMU on the system. 
- Select "Install JumpCloud Agent" and migrate the AD user to a new local user account
- The tool should not throw an error as it downloads the prerequisite files 
- The tool should also not throw an error as it encounters the AppxPackage step
- Upon login to the new local user account, the migration should complete with the JumpCloud logo

## Screenshots
<img width="929" alt="Screen Shot 2022-10-14 at 2 04 30 PM" src="https://user-images.githubusercontent.com/54448601/197577170-30b1e50e-cc77-4aaf-9da3-11192b784b4a.png">
<img width="1023" alt="Screen Shot 2022-10-14 at 2 04 41 PM" src="https://user-images.githubusercontent.com/54448601/197577177-9631edf5-5b8d-42fe-83ea-e56d22ef1aa1.png">
